governator-core/src/main/java/com/netflix/governator/InjectorBuilder.java

```diff
@@ -62,6 +63,23 @@ public static InjectorBuilder fromModules(List<Module> modules) {
         return new InjectorBuilder(Modules.combine(modules));
     }
     
+    public static <T> InjectorBuilder bindInstance(final Object o) {
```

Comment:
moved logic out of InjectorBuilder; now looks like this:

        try (LifecycleInjector injector = TestSupport.inject(postConstructChild)) {
           [ ...]
        }